src/Service/DataFetcher.php

```diff
@@ -34,36 +26,27 @@ public function __construct(
         $this->dateConversion = $dateConversion;
     }
 
-    public function fillingTable(): void
+    public function fillingTable(array $currencies): void
     {
         if ($this->btcCourseRepository->isEmptyTable() === true) {
```

Comment:
```suggestion
        if ($this->btcCourseRepository->isEmptyTable()) {
```

можна не додавати перевірку `=== true` якщо у тебе метод повертає одразу `true` і має назву таку, що зрозуміло що він повертає bool. тут з назвою все ок

Reply:
```suggestion
        if ($this->btcCourseRepository->isEmpty() === true) {
```